storm-client/src/jvm/org/apache/storm/StormSubmitter.java

```diff
@@ -126,7 +127,7 @@ public static void pushCredentials(String name, Map<String, Object> topoConf, Ma
      * @throws NotAliveException        if the topology is not alive
      * @throws InvalidTopologyException if any other error happens
      */
-    public static void pushCredentials(String name, Map<String, Object> topoConf, Map<String, String> credentials, String expectedUser)
+    public static boolean pushCredentials(String name, Map<String, Object> topoConf, Map<String, String> credentials, String expectedUser)
```

Comment:
missing javadoc `@return`

There is another `pushCredentials` method in this class needs to be taken care of

Reply:
Fixed